index.html

```diff
@@ -785,6 +788,51 @@
           "site preferences".
         </p>
       </section>
+      <section>
+        <h3>
+          “Nice to have” images
+        </h3>
+        <p>
+          Images are often content, but there are instances when an image is an
+          enhancement to the document, but not a necessary part of the core
+          content (a.k.a. “nice to have“). As such, Authors may choose to load
+          an image only above a certain breakpoint in order to
+          optimize the readbility of the core content at certain sizes by
+          reducing the visual clutter. This is a similar use case to <a href="#art-direction">Art direction</a>, but distinct enough to warrant separation.
```

Comment:
To me, that seems extremely fragile. It would be super easy to screw up the breakpoints because developers making assumptions about what is "below the fold"  or "outside the view", when on different screen sizes it might not apply. 

I'm probably missing something, so will let @yoavweiss chime in and tell me how wrong I am :)